.storybook/main.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
-  stories: ['../src/**/*.stories.js'],
-  addons: [
-    '@storybook/preset-create-react-app',
-    '@storybook/addon-actions',
-    '@storybook/addon-links',
-  ],
-};
+    stories: ['../src/**/*.stories.js'],
+    addons: [
+      '@storybook/preset-create-react-app',
+      '@storybook/addon-actions',
+      '@storybook/addon-links',
+    ],
+  };
```

Comment:
Can you cleanup these changes.

Reply:
done closing this pull request and going to add a new pull request.